src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -1211,4 +1211,14 @@ public float getBlockHardness() {
     public float getBlockResistance() {
         return 6.0f;
     }
-}
\ No newline at end of file
+
+    /**
+     * Override this if the MTE will keep its Item inventory on-break
+     * If this is carelessly overridden and set to true, then Items will be voided.
```

Comment:
I think this should be more informative that by setting this to true author is responsible for handling inventory storing and restoring on break placement respectively